src/options.object.ts

```diff
@@ -10,5 +10,5 @@ export type OptionsObject = {
   keepImageTypes?: string[];
   ignoreImageTypes?: string[];
   quality?: number;
-  insertIntoEditor?: (base64URL: string, imageBlob: Blob, editor: Quill) => void;
+  uploadImage?: (imageBlob: Blob) => Promise<string>;
```

Comment:
Like you mentioned, this is a breaking change to the API, this would mean we'd need to release a new major version, which I don't think this change warrants.

At the very least, the current `insertIntoEditor` option should probably be maintained and marked as `@deprecated`

Reply:
I agree partiallay, but unfortunately we are not able to support this method when html is pasted, due to its method signature. 
My thought was to better have a breaking change than to have a legacy method which is only partially supported.

Reply:
> but unfortunately we are not able to support this method when html is pasted, due to its method signature.

Can we at least support it for non-html pastes then? 

I'd really rather keep the API backwards compatible, as personally I hate it when libraries rename their API methods...

Reply:
Well I'm the complete opposite:  I rather like my code to fail fast instead of having an unexpected behavior that is lurking on spacial conditions. 

But it is your repository, so it is your decision to change it or not.